package exception;

public class exceptionDemo {
	public static void main(String[] args) {
		// TODO Auto-generated method stub
		System.out.println("The exception we got is");
		divis.divide();
	}
}
